ingestr/src/sources.py

```diff
@@ -480,3 +481,35 @@ def dlt_source(self, uri: str, table: str, **kwargs):
         return hubspot(
             api_key=api_key[0],
         ).with_resources(endpoint)
+
+
+class AirtableSource:
+    def handles_incrementality(self) -> bool:
+        return True
+
+    # airtable://?access_token=<access_token>&base_id=<base_id>
+
+    def dlt_source(self, uri: str, table: str, **kwargs):
+        if kwargs.get("incremental_key"):
+            raise ValueError(
+                "Airtable takes care of incrementality on its own, you should not provide incremental_key"
+            )
```

Comment:
is that actually the case? it seems like Airtable does not support incremental loading, therefore we should return an error that says incremental loading is not supported for Airbyte.

Please also document the same in the docs, similar to how Google Sheets source is documented.

Reply:
Yes, it doesn’t support incremental loading. I have made necessaries changes.


<img width="565" alt="Screenshot 2024-08-29 at 13 00 16" src="https://github.com/user-attachments/assets/cfbf4132-0fc5-42c8-9849-afe0574c1bcc">
<img width="1072" alt="Screenshot 2024-08-29 at 13 09 18" src="https://github.com/user-attachments/assets/0d55a3ae-e50e-4c6e-9286-ab9674794750">